includes/alerts/transport.opsgenie.php

```diff
@@ -0,0 +1,45 @@
+/* Copyright (C) 2017 Celal Emre CICEK <celal.emre@opsgenie.com>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. */
+
+/**
+ * OpsGenie API Transport
+ * @author Celal Emre CICEK <celal.emre@opsgenie.com>
+ * @copyright 2017 Celal Emre CICEK
+ * @license GPL
+ * @package LibreNMS
+ * @subpackage Alerts
+ */
+
+$url = $opts['url'];
+
+$curl = curl_init();
+
+set_curl_proxy($curl);
+curl_setopt($curl, CURLOPT_URL, $url );
+curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "POST"); 
+curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+curl_setopt($curl, CURLOPT_HTTPHEADER, array('Content-Type: application/json'));
+curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($obj));
+
+$ret = curl_exec($curl);
+$code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+
+var_dump("Response from OpsGenie: " . $ret); //FIXME: proper debugging
```

Comment:
Is this left in by accident?

Reply:
Do you mean the FIXME part? If so, no. I left it there, because after inspecting other parts of the code, I saw these kind of FIXMEs next to all the var_dump statement, and thought if a more proper debugging mechanism is introduced in the future, then someone can apply it to my code, too. If you mean the var_dump statement itself not being in the if statement below it, that's also on purpose. We would like to be able to have the response from OpsGenie even if the request is made successfully. But, I can move it into the if statement to have the response from OpsGenie only when an error occurs, if it's more acceptable that way.

Reply:
I meant the FIXME one. That's going to be output no matter what so should be removed imho. The one in the != 200 if statement is fine though.

Reply:
I see. I'll move it into the if statement below, then. Thank you!